src/askscript/__tests__/07-arrays/program17e-object_has_key_is_false.ask

```diff
@@ -0,0 +1,9 @@
+ask {
+  const countries = {usa: 'USA', spain: 'Spain', russia: 'Russia'};
+  const sweden = 'Sweden';
+  if(!(countries:hasKey(sweden:toLowerCase))) {
```

Comment:
I would aim for creating a minimal working tests, so I'd remove `const sweden = 'Sweden';` and replaced `sweden:toLowerCase` with just `'sweden'`.

Reply:
But it's fine with the code you wrote.